build.gradle

```diff
@@ -66,8 +65,15 @@ compileTestJava {
 }
 
 repositories {
+    mavenCentral()
     mavenLocal()
     jcenter()
+    mavenCentral()
+    maven { url 'https://jitpack.io' }
+
+    maven {
+        url 'https://repo.spring.io/libs-milestone'
```

Comment:
why is this now needed?

Reply:
Referring to the notes on the slack channel and the example. Seems we need these now. 
https://github.com/hmcts/sscs-common